.github/ISSUE_TEMPLATE/Feature.md

```diff
@@ -7,9 +7,16 @@ about: Suggest a new feature or enhancement for rclone
 
 Welcome :-)
 
-So you've got an idea to improve rclone? We love that! You'll be glad to hear we've incorporated hundreds of ideas from contributors already.
+So you've got an idea to improve rclone? We love that!
+You'll be glad to hear we've incorporated hundreds of ideas from contributors already.
+Probably the latest release has your feature. Try to update your rclone, run:
 
-Here is a checklist of things to do:
+    rclone self-update
+
+If your rclone reports `unknown command "self-update"` then you'll need to update manually
+following the install instructions at https://rclone.org/install
+
+If it still isn't there, here is a checklist of things to do:
 
   1. Please search the old issues first for your idea and +1 or comment on an existing issue if possible.
   2. Discuss on the forum first: https://forum.rclone.org/
```

Comment:
Good idea to also specify the template to be used in the forum i.e. `Feature` or `Dev Discussion` depending on the type of request and the level of involvement the user wants to have.

Reply:
Can we leave this to the forum template?

Reply:
Fine either way but if we are not mentioning the template here then we should probably remove the mention of the `Help & Support Template` from the Bug template too.